src/puppetlabs/jdbc_util/pglogical.clj

```diff
@@ -123,11 +123,12 @@
   or :none if replication-mode is :none. Valid status values vary depending on
   thew role; see other replication-status functions for more information."
   [db replication-mode]
-  (case replication-mode
-    :source (provider-replication-status db)
-    :replica (replica-replication-status db)
-    :none :none
-    nil :none))
+  (with-timeout 4 :unknown
```

Comment:
@holguinj do you know if we have any baseline for how long these queries should take?

@jonathannewman I'm not sure exactly how long these queries can take, not sure how to establish a baseline for that (although the max we can have here is something like 30s otherwise tk-status will timeout).